.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: node_js
+sudo: false
+node_js:
+  - '0.10'
+  - '0.12'
+  - stable
+
+env:
+  global:
+    - secure: CxEg2hSOUgwATFFAbD5ycalZdVvaKSTGIVyVYGze7NCqNaDhaXr1VSfrdfu+GrPp6+cYPhBB8Rr0LaweyZPqiVEHFWBtUiOjl5PShOazyiCNTX6ZQUCTHYQVkoIyKYhP/WVzmy6MfhCnuSkA129EeQ6dVUr/8NlIW2/N8PgU018=
+    - GH_OWNER: zckrs
+    - GH_PROJECT_NAME: gulp-angular-boilerplate
```

Comment:
I'm using this yeoman project https://github.com/Swiip/generator-gulp-angular to develop such kind of apps. This file was needed only for initiating seed with desired techs. I'm removing it now.